src/App.tsx

```diff
@@ -1,18 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+// import { event } from 'cypress/types/jquery';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42).map(n => `Item ${n}`);
+
+enum ItemsPerPage {
+  three = 3,
+  five = 5,
+  ten = 10,
+  twenty = 20,
+}
```

Comment:
```suggestion
enum ItemsPerPage {
  Three = 3,
  Five = 5,
  Ten = 10,
  Twenty = 20,
}
```
use CamelCase for enum keys
:+1: for enum :)